File: modulo1/segmento3/apunte7Funciones/Ejercicio5.java
import java.util.Scanner;

public class Ejercicio5{

	public static Integer cubo(Integer num){
		Integer total=num;

		for(Integer i=0;i<=1;i++){
			total*=num;
		}

		return total;
	
	
	}
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);
		Integer total;

		System.out.println("Ingrese un numero: ");
		Integer num= Integer.parseInt(teclado.nextLine());

		total= cubo(num);

		System.out.println("El total es: "+ total );
	
	}

}

File: modulo1/segmento3/apunte7Funciones/ejercicio5.py
def cubo(num1):
	result=num1
	for i in range(1,3):
		result*=num1
	return result

num1=int(input("Ingrese un numero: " ))

respuesta= cubo(num1)

print(str(num1) +" elevado al cubo es igual a: " + str(respuesta))
		
